src/lua/serializer.c

```diff
@@ -76,12 +76,32 @@ static struct {
 	OPTION(LUA_TBOOLEAN, encode_use_tostring, 0),
 	OPTION(LUA_TBOOLEAN, encode_invalid_as_nil, 0),
 	OPTION(LUA_TBOOLEAN, encode_error_as_ext, 1),
+	OPTION(LUA_TBOOLEAN, encode_esc_slash, 0),
 	OPTION(LUA_TBOOLEAN, decode_invalid_numbers, 1),
 	OPTION(LUA_TBOOLEAN, decode_save_metatables, 1),
 	OPTION(LUA_TNUMBER,  decode_max_depth, 128),
 	{ NULL, 0, 0, 0},
 };
 
+/** toggler for json.encode behavior change with tarantool.compat (gh-7000) */
```

Comment:
Nit: We start a comment from a capital letter and end it with a period. Please, adjust according to the style over the whole patch.

Reply:
Isn't that just for Lua code style?

Reply:
Fixed anyway.